modules/extension/xsd/xsd-sld/src/test/java/org/geotools/sld/v1_1/SLDExampleTest.java

```diff
@@ -192,6 +192,11 @@ public void testParseValidateVendorOptions() throws Exception {
         
         // check it passes validation
         List errors = validate(file);
+        if(errors.size()>0) {
+            for(int i=0;i<errors.size();i++) {
+                System.out.println(errors.get(i));
+            }
```

Comment:
On Thu, 6 Aug, 2015 at 2:28 PM, Daniele Romagnoli 
<notifications@github.com> wrote:
> In 
> modules/extension/xsd/xsd-sld/src/test/java/org/geotools/sld/v1_1/SLDExampleTest.java:
> 
> > @@ -192,6 +192,11 @@ public void testParseValidateVendorOptions() 
> throws Exception {
> >          
> >          // check it passes validation
> >          List errors = validate(file);
> > +        if(errors.size()>0) {
> > +            for(int i=0;i<errors.size();i++) {
> > +                System.out.println(errors.get(i));
> > +            }
> Is it really needed? (Wondering if it has been added for 
> debugging/testing purposes and forgotten)
> 
Well spotted that is from a debugging session. I'll push a fix.

Ian
> —
> 
> Reply to this email directly or view it on GitHub.
>